.github/workflows/cd-pipeline.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         with:
           path: charts
           fetch-depth: 2 # to be able to obtain files changed in the latest commit
+          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
```

Comment:
Why is `BITNAMI_BOT_TOKEN` required? This could trigger other workflows, is that expected? If you need write access to the repository you can change the `permissions` section int the job definition. https://github.com/bitnami/charts/blob/ef3fea44075841d44274d1c26c3723e5beec0e3c/.github/workflows/cd-pipeline.yml#L18-L19

Reply:
We need write access to push the tags